pkg/cmd/roachtest/roachtestutil/mixedversion/testdata/planner/separate_process

```diff
@@ -29,181 +29,226 @@ ok
 plan debug=true
 ----
 Seed:               12345
-Upgrades:           v21.2.11 → v22.1.8 → v22.2.3 → v23.1.4 → v23.2.0 → <current>
```

Comment:
Pitfall of data driven tests, this was broken the entire time but easy to miss when the diffs are so large.

Reply:
Hmm, on second thought, now that this is fixed it also means this test will flake every time there's a new release.